lib/dgram.js

```diff
@@ -639,6 +639,25 @@ Socket.prototype.unref = function() {
   return this;
 };
 
+
+Socket.prototype.setRecvBufferSize = function(size) {
+  var err = this._handle.setRecvBufferSize(size);
+
+  if (err) {
+    throw errnoException(err, 'setRecvBufferSize');
```

Comment:
Would prefer using `internal/errors` here since this introduces new errors. The errno can be added.